kythe/rust/extractor/rustfmt.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
Is there a reason this file (and its config) is in kythe/rust/extractor? A better place might be `/tools/rust`.

Reply:
That's true. I have created a universal script in `/tools/rust` and removed the scripts from the extractor and indexer directories